pandas/core/series.py

```diff
@@ -5745,7 +5745,49 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0,
+        doc="""
+        The index (axis labels) of the Series.
+        Get the index (row labels) of the Series.
+
+        Returns
+        -------
+        Index
+            The index labels of the Series.
+
+        See Also
+        --------
+            Series.reindex:
+                Conform Series to new index.
+            Series.set_index:
+                Set Series as DataFrame index.
+            Index:
+                The base pandas index type.
+
+        Notes
+        -----
+        The index of a Series is used to label and identify each element of the
+        underlying data. The index can be thought of as an immutable ordered set
+        (technically a multi-set, as it may contain duplicate labels), and is
+        used to index and align data in pandas.
+
+        For more information on pandas indexing, see the `indexing user guide
+        <https://pandas.pydata.org/docs/user_guide/indexing.html>`__.
+        Examples
+        --------
+        To create a Series with a custom index and view the index labels:
+
+        >>> cities = ['Kolkata', 'Chicago', 'Toronto', 'Lisbon']
+        >>> populations = [14.85, 2.71, 2.93, 0.51]
+        >>> city_series = pd.Series(populations, index=cities)
+        >>> city_series.index
+        Index(['Kolkata', 'Chicago', 'Toronto', 'Lisbon'], dtype='object')
+
+        To change the index labels of an existing Series:
+        >>> city_series.index = ['KOL', 'CHI', 'TOR', 'LIS']
```

Comment:
I think we need a blank line here

```suggestion

        To change the index labels of an existing Series:
        >>> city_series.index = ['KOL', 'CHI', 'TOR', 'LIS']
```

Currently, it is rendered like

![image](https://user-images.githubusercontent.com/6849766/230618476-c836a590-ddb2-402e-bca2-f08c02baf5cb.png)